src/code-block/transforms.js

```diff
@@ -21,6 +21,16 @@ export default {
 				node.children.length === 1 &&
 				node.firstChild.nodeName === 'CODE'
 			),
+			transform( node ) {
+				const content = node.firstChild.textContent;
+				const [ startingMatch, language ] = content.match( /^```(\w+)?\n/ ) || [ null, null ];
+				const attributes = language ? { language } : {};
+
+				// Extract content without backticks and (optionally) language.
+				attributes.content = startingMatch ? content.replace( startingMatch, '' ).replace( /```$/, '' ) : content;
```

Comment:
The closing backticks were not removed when some whitespace gets in there — which also seems to happen when more than a single block is pasted. To test, I just copy-pasted the entire description from this PR as one.

Allowing some whitespace in the regexp seems to fix it:

```suggestion
				attributes.content = startingMatch ? content.replace( startingMatch, '' ).replace( /```\s*$/, '' ) : content;
```